wagtailio/project_styleguide/templates/patterns/includes/search.html

```diff
@@ -0,0 +1,23 @@
+<div class="modal modal--search micromodal-slide" id="site-search-modal" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1" data-micromodal-close>
+        <div class="modal__container" role="dialog" aria-modal="true">
+            <main class="modal__content">
+                <button class="modal__close-button" aria-label="Close the Search" data-micromodal-close>
```

Comment:
https://user-images.githubusercontent.com/22497837/183664720-c4bcadc3-d22e-424a-bd81-19f17f39cd44.mp4

I think the offset makes the onclick of the close a little inconsistent, is there anything we can do about this?

Reply:
Great spot. Removing the pointer events from the icon seems to remedy this.